src/containers/auth/login/index.js

```diff
@@ -1,22 +1,35 @@
-import React, { Component, Fragment } from 'react';
+import React, { Fragment } from 'react';
 import Modal from 'react-modal';
 import { connect } from 'react-redux';
 import { isEmpty } from 'lodash';
 import { FaTimes } from 'react-icons/fa';
-import GoogleLogin from 'react-google-login';
-import { loginRequest, googleLoginRequest } from '../../../actions/auth';
+import ReactQueryParams from 'react-query-params';
+import {
```

Comment:
Fix imports order: base react libraries - custom react libraries - your components - styles. Separate every group by whitespace. Check it in all your files